recipes/libmad/all/conanfile.py

```diff
@@ -23,9 +23,17 @@ def config_options(self):
     def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
-        if self._is_msvc and self.options.shared:
-            raise ConanInvalidConfiguration("libmad does not support shared library for MSVC")
-
+ 
+    def validate(self):
+        if self.options.shared:
+            del self.options.fPIC
```

Comment:
This goes in `configure`, but the check in `validate` 😄

Reply:
🤦‍♂️